docs/kubernetes/user-guides/pre-provisioned.md

```diff
@@ -0,0 +1,86 @@
+# Kubernetes Pre-Provisioned Disks User Guide
+This guide gives a simple example on how to use this driver with disks that have been pre-provisioned by an administrator.
+
+## Install Driver
+
+See [instructions](driver-install.md)
+
+## Pre-Provision a Disk
+
+If you have not already pre-provisioned a disk on GCP you can do that now.
+
+1. Create example PD
+
+```bash
+gcloud compute disks create test-disk --zone=us-central1-c --size=5Gi
+```
+
+## Create Persistent Volume for Disk
+
+1. Create example Zonal Storage Class
+
+```bash
+kubectl apply -f ./examples/kubernetes/demo-zonal-sc.yaml
+```
+
+2. Create example Persistent Volume
+
+**Note:** The `volumeHandle` and `nodeSelectorTerms` values should be generated
+based on the project, zone, and PD name of the disk created.
+
+**RePD Note:** For RePD the volume handle is of the form:
```

Comment:
Maybe have explicit examples for zonal vs repd?

Reply:
I would have to write a very similar note anyway about modifying these specific parts of the spec vs. right now which is modify + add one little thing. I think this is clear enough and reduces duplication

Reply:
Would it be easier to spot the difference if you had the 2 formats side by side in the spec?

Reply:
not sure what you mean by "side by side" in this case. I do feel like it could be clearer but I'm not sure that duplicating everything for RePD is the solution. It feels a little clumsy (especially in terms of keeping in sync when things change).

Reply:
Like maybe embed the RePD syntax as a comment in the spec above the zonal syntax line? Kind of like how the 2nd zone is also embedded into the spec too.

Also we should stick to the official "Regional PD" name.